niri-config/src/lib.rs

```diff
@@ -371,14 +377,67 @@ pub struct Tablet {
     pub left_handed: bool,
 }
 
-#[derive(knuffel::Decode, Debug, Default, PartialEq)]
+static DEFAULT_TABLET: LazyLock<Tablet> = LazyLock::new(Tablet::default);
+
+impl FromIterator<Tablet> for Tablets {
+    fn from_iter<T: IntoIterator<Item = Tablet>>(iter: T) -> Self {
+        Self(Vec::from_iter(
+            iter.into_iter().filter(|t| *t != *DEFAULT_TABLET),
```

Comment:
Why this check? It seems like having them there is redundant, but also doesn't necessarily hurt?